src/ui/NoteSelectWindow/index.js

```diff
@@ -0,0 +1,34 @@
+setTimeout(init, 5, 10);
+
+function init() {
+    const notesList = JSON.parse(document.getElementById("notesList").value);
+
+    let notesObject = {};
+    let orderedNotebooks = [];
```

Comment:
Instead of `orderedNotebooks` you can use [notesObject.keys()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys)

Reply:
Ooo that's a very useful function! Using it now :)